src/BlockInfo.cpp

```diff
@@ -584,6 +584,27 @@ void cBlockInfo::Initialize(cBlockInfoArray & a_Info)
 	a_Info[E_BLOCK_STONE               ].m_CanBeTerraformed = true;
 
 
+	// Block heights:
+	a_Info[E_BLOCK_BED                 ].m_BlockHeight = 9.0 / 16.0;
+	a_Info[E_BLOCK_CAKE                ].m_BlockHeight = 8.0 / 16.0;
+	a_Info[E_BLOCK_ENCHANTMENT_TABLE   ].m_BlockHeight = 12.0 / 16.0;
+	a_Info[E_BLOCK_STONE_SLAB          ].m_BlockHeight = 0.5;
+	a_Info[E_BLOCK_WOODEN_SLAB         ].m_BlockHeight = 0.5;
+	a_Info[E_BLOCK_SNOW                ].m_BlockHeight = 1.0 / 8.0;
+	a_Info[E_BLOCK_ACACIA_FENCE        ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_ACACIA_FENCE_GATE   ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_BIRCH_FENCE         ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_BIRCH_FENCE_GATE    ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_DARK_OAK_FENCE      ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_DARK_OAK_FENCE_GATE ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_FENCE               ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_OAK_FENCE_GATE      ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_JUNGLE_FENCE        ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_JUNGLE_FENCE_GATE   ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_SPRUCE_FENCE        ].m_BlockHeight = 1.5;
+	a_Info[E_BLOCK_SPRUCE_FENCE_GATE   ].m_BlockHeight = 1.5;
```

Comment:
My vote is for floats / doubles, too. I'd prefer using fractions (`1.0 / 16.0`) to represent them, rather than decimal numbers, since fractions are somewhat easier to read in this case - most, if not all, of the numbers will be `N / 16.0`.